_includes/icons/preconfigured.svg

```diff
@@ -0,0 +1 @@
+server.svg
```

Comment:
I would probably recommend using something different than server.svg, as it does have a different size than other icons used for variants

Reply:
I did re-use kvm icon, however I've openned #77